source/fe/fe_nedelec_sz.cc

```diff
@@ -211,46 +236,67 @@ FE_NedelecSZ<dim, spacedim>::shape_grad_grad(const unsigned int /*i*/,
 template <int dim, int spacedim>
 Tensor<2, dim>
 FE_NedelecSZ<dim, spacedim>::shape_grad_grad_component(
-  const unsigned int /*i*/,
-  const Point<dim> & /*p*/,
-  const unsigned int /*component*/) const
+  const unsigned int i,
+  const Point<dim>  &p,
+  const unsigned int component) const
 {
-  DEAL_II_NOT_IMPLEMENTED();
-  return Tensor<2, dim>();
+  AssertIndexRange(i, this->n_dofs_per_cell());
+  AssertIndexRange(component, dim);
+
+  std::unique_ptr<
+    typename dealii::FiniteElement<dim, spacedim>::InternalDataBase>
+    data_ptr = std::make_unique<InternalData>();
+
+  std::vector<Point<dim>> p_list = {p};
+
+  // compute the data
+  this->evaluate(p_list, update_hessians, data_ptr);
+
+  // access the data
+  auto &data = dynamic_cast<InternalData &>(*data_ptr);
+  return data.shape_hessians[i][0][component];
 }
 
 
 
 template <int dim, int spacedim>
-std::unique_ptr<typename dealii::FiniteElement<dim, spacedim>::InternalDataBase>
-FE_NedelecSZ<dim, spacedim>::get_data(
-  const UpdateFlags update_flags,
-  const Mapping<dim, spacedim> & /*mapping*/,
-  const Quadrature<dim> &quadrature,
-  dealii::internal::FEValuesImplementation::FiniteElementRelatedData<dim,
-                                                                     spacedim>
-    & /*output_data*/) const
-{
+void
+FE_NedelecSZ<dim, spacedim>::evaluate(
+  const std::vector<Point<dim>> /*p_list*/,
+  const UpdateFlags /*update_flags*/,
   std::unique_ptr<
     typename dealii::FiniteElement<dim, spacedim>::InternalDataBase>
-        data_ptr   = std::make_unique<InternalData>();
+    & /*data_ptr*/) const
+{
+  DEAL_II_NOT_IMPLEMENTED();
+}
+
+
+
+template <>
+void
+FE_NedelecSZ<2, 2>::evaluate(
+  const std::vector<Point<2>> p_list,
+  const UpdateFlags           update_flags,
+  std::unique_ptr<typename dealii::FiniteElement<2, 2>::InternalDataBase>
+    &data_ptr) const
```

Comment:
The function used to be implemented for arbitrary dimensions, but here you restrict it to `dim==2`. Did we ever call the `dim!=2` case before?